diff --git a/node_modules/react-native-worklets/android/CMakeLists.txt b/node_modules/react-native-worklets/android/CMakeLists.txt
--- a/node_modules/react-native-worklets/android/CMakeLists.txt
+++ b/node_modules/react-native-worklets/android/CMakeLists.txt
@@ -88,7 +88,7 @@
 # build shared lib
 set_target_properties(worklets PROPERTIES LINKER_LANGUAGE CXX)
 
 target_link_libraries(worklets log ReactAndroid::reactnative ReactAndroid::jsi
-                      fbjni::fbjni)
+                      fbjni::fbjni c++_shared)
 
 if(${JS_RUNTIME} STREQUAL "hermes")
